### DISCUSSION TITLE
how to install ext-ldap

### DISCUSSION THREAD

**Discussion by fey** [asdf-community/asdf-php#76] [Q&A] [answered]
Hello, how i can insall extensions to php installed via asdf?

``` 
$ asdf version
v0.7.8
```
macos 10.14.5

**Comment by smorimoto** [comment, marked as answer]
Here is not a support forum. Also, you can do it with reference to this page https://www.php.net/manual/en/book.ldap.php and a similar past issue https://github.com/asdf-community/asdf-php/issues/65.
If you read it and still don't know how to build it, we need your detailed environment information in the first place.

**Comment by fey** [comment]
Thanks, i will check it
